app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -228,6 +228,23 @@
   </section>
 <% end %>
 
+
+<% if !@is_enhanced_ipp %>
+  <h3><%= t('in_person_proofing.body.location.change_location_heading') %></h3>
+  <p class="margin-bottom-4">
+    <%= t(
+      'in_person_proofing.body.location.change_location_info',
+      find_other_locations_link: link_to(
+        t('in_person_proofing.body.location.change_location_find_other_locations'),
+        help_center_redirect_url(
+          category: 'verify-your-identity',
+          article: 'verify-your-identity-in-person/find-a-participating-post-office',
+        )
+      )
+    ).html_safe %>
```

Comment:
Instead of `html_safe`, I suggest using `_html` suffix convention for translated strings:

https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
Thanks!! Will make this update.